README.md

```diff
@@ -10,7 +10,7 @@ to simplify building and publishing Kubernetes APIs from scratch.
 
 ### Installalation 
 
-It is strongly recommended that you use a released version. Release binaries are available on the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page.
+It is strongly recommended that you use a released v1 version, since v0 release is deprecated and not supported. v1 has different design and implements, which is not compatible with v0. Release binaries are available on the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page.
```

Comment:
hmm... I don't think new users are downloading the `old` (v0) releases because by default release pages shows the v1.0.x release, so adding text here is not going to be of much help IMO. We can add a note at the bottom of this section about the deprecating of v0. 

One place where messaging can help would be: if people specify `v0` project while initializing the project. Adding message there and in the help text would useful.